tests/UI/campaigns/functional/FO/09_productPage/02_addToCart.ts

```diff
@@ -185,8 +185,8 @@ describe('FO - product page : Add product to cart', async () => {
 
         // Create products images
         await Promise.all([
-          files.generateImage(productData.coverImage),
-          files.generateImage(productData.thumbImage),
+          files.generateImage(productData.coverImage ?? ''),
+          files.generateImage(productData.thumbImage ?? ''),
```

Comment:
Why `productData.coverImage` could be undefined? It looks like it's always defined

Reply:
Because it's not always defined with `Product` type.